experiment/slack-oncall-updater/main.go

```diff
@@ -26,6 +26,31 @@ import (
 	"strings"
 )
 
+// githubToSlack maps (lowercase!) GitHub usernames to Slack user IDs.
+// Add yourself when you join the oncall rotation.
+// You can mostly easily find your user ID by visiting your Slack profile, clicking "...",
+// and clicking "Copy user ID".
+var githubToSlack = map[string]string{
+	"amwat":          "U9B1P2UGP",
+	"bentheelder":    "U1P7T516X",
+	"chases2":        "UJ9R0FWD6",
+	"cjwagner":       "U4QFZFMCM",
+	"fejta":          "U0E2KHQ13",
+	"katharine":      "UBTBNJ6GL",
+	"michelle192837": "U3TRY5WV7",
+	"mushuee":        "U013TPFJWC8",
+	"spiffxp":        "U09R2FL93",
+}
+
+// rotationToSlack group maps the rotation in go.k8s.io/oncall to the slack
+// group ID for the oncall alias group for this rotation
+var rotationToSlackGroup = map[string]string{
+	"testinfra":          "SGLF0GUQH",   // @test-infra-oncall
+	"google-build-admin": "S017N31TLNN", // @google-build-admin
```

Comment:
This should also be plural `"google-build-admins":"bentheelder"`

Reply:
cl/322878818 should take effect at 4:27

Reply:
it didn't take effect yet, so I added a second entry for both.